examples/web/src/features/json/browser/styles.css

```
@@ -1,33 +1,33 @@
-    * {
+    .json-surface, .json-surface * {
       box-sizing: border-box;
       margin: 0;
       padding: 0;
     }
 
-    body {
+    .json-surface {
       font-family: 'Consolas', 'Monaco', 'Courier New', monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)